tests/core_tests.py

```diff
@@ -199,11 +199,16 @@ def test_annotation_json_endpoint(self):
         self.assertIn("id", resp_annotations["result"][0])
         self.assertIn("name", resp_annotations["result"][0])
 
-        layer = self.get_resp(
+        response = self.get_resp(
             f"/superset/annotation_json/{layer.id}?form_data="
             + quote(json.dumps({"time_range": "100 years ago : now"}))
         )
-        self.assertIn("my_annotation", layer)
+        self.assertIn("my_annotation", response)
```

Comment:
nit, while at it, perhaps do
```suggestion
        assert "my_annotation" in response
```